docs/content/latest/explore/multi-region-deployments/row-level-geo-partitioning.md

```diff
@@ -180,20 +180,20 @@ First, we create the parent table that contains a `geo_partition` column which i
 
     ```output
                     List of relations
-     Schema |         Name         | Type  |  Owner
-    --------+----------------------+-------+----------
-    public | transactions         | table | yugabyte
-    public | transactions_eu      | table | yugabyte
-    public | transactions_india   | table | yugabyte
-    public | transactions_us      | table | yugabyte
+     Schema |         Name                      | Type  |  Owner
+    --------+-----------------------------------+-------+----------
+     public | bank_transactions         | table | yugabyte
+     public | bank_transactions_eu      | table | yugabyte
+     public | bank_transactions_india   | table | yugabyte
+     public | bank_transactions_us      | table | yugabyte
     (4 rows)
     ```
 
 The data is now arranged as follows:
 
 ![Row-level geo-partitioning](/images/explore/multi-region-deployments/geo-partitioning-2.png)
 
-## Step 3. Pinning user transactions to geographic locations
+## Step 3. Pinning user bank transactions to geographic locations
```

Comment:
Pinning user partitions specific to the geographic locations.

Reply:
In this sub-section, please specify which cluster the user needs to connect to before running the inserts. We talk about the force_global_transaction flag in section 5 below. At this point, its better to specify explicit connection aspects, so that an user would be able to run the examples successfully.

Reply:
re: specifying connection, this is a fast path example so it's not needed to run the examples; this is explained in section 5.